### DISCUSSION TITLE
[Curso: TodoAPI com ASPNET, CQRS e EFCORE] - Dúvidas com a nova documentação do Flunt.

### DISCUSSION THREAD

**Discussion by lucascidade** [balta-io/forum#429] [Tira Dúvidas] [answered]
Olá, pessoal, boa noite!

Estou com uma dúvida quando aos contratos dentro do Flunt, no curso temos o código

```
AddNotifications(new Contract()
.Requires()
.HasMinLen(Title, 3, "Title", "Por favor, descrever .... "
....
```

Quanto ao uso do HasMinLen, pelo o que eu vi na documentação atual este atributo não existe mais, qual seria o melhor substituto dele nesse caso? IsGreaterThan? IsLowerThan?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @lucascidade, como vai?
Na versão 2.x do Flunt, alguns contratos mudaram, por favor, utilize a mesma versão do curso:
```
dotnet add package Flunt --version 1.0.5
```
PS:. Substituir a versão informada acima pela apresentada no curso.

Duplicado:
https://github.com/balta-io/forum/discussions/235
